src/api/commentReply/sendCommentReply.ts

```diff
@@ -0,0 +1,16 @@
+import { API, handleAPIResponse } from '@/api/api';
+import { StatusCodes } from 'http-status-codes';
+import { CommentReply } from '@/api/commentReply/commentReply.interface';
+
+export interface SendReplyRequestDto {
+  body: string;
+}
+
+export async function sendCommentReply(commentId: string, body: string): Promise<CommentReply | null> {
+  const res = await API.post<SendReplyRequestDto, CommentReply>(`/ue/comments/${commentId}/reply`, { body });
+  let reply: CommentReply | null = null;
+  handleAPIResponse(res, {
+    [StatusCodes.CREATED]: (body) => (reply = body),
+  });
+  return reply;
```

Comment:
Sauf si ça pose particulièrement problème
```suggestion
  return handleAPIResponse(res, {
    [StatusCodes.CREATED]: (body) => body,
  });
```

Reply:
Nope, juste que je pense que c'est parce que le return type était nouveau, donc j'ai pas du remplacer partout